src/palace/manager/core/config.py

```diff
@@ -72,46 +71,26 @@ class Configuration(ConfigurationConstants):
 
     @classmethod
     def database_url(cls):
-        """Find the database URL configured for this site.
-
-        For compatibility with old configurations, we will look in the
-        site configuration first.
-
-        If it's not there, we will look in the appropriate environment
-        variable.
-        """
-
-        # To avoid expensive mistakes, test and production databases
-        # are always configured with separate keys. The TESTING variable
-        # controls which database is used, and it's set by the
-        # package_setup() function called in every component's
-        # tests/__init__.py.
-        test = os.environ.get("TESTING", False)
-        if test:
-            environment_variable = cls.DATABASE_TEST_ENVIRONMENT_VARIABLE
-        else:
-            environment_variable = cls.DATABASE_PRODUCTION_ENVIRONMENT_VARIABLE
-
-        url = os.environ.get(environment_variable)
```

Comment:
Setting up the DB for testing is entirely handled by the pytest mocks now and doesn't depend on the `TESTING` env var.